app/src/test/java/com/samco/trackandgraph/antlr/DatapointGen.kt

```diff
@@ -0,0 +1,49 @@
+package com.samco.trackandgraph.antlr
```

Comment:
I think i copied this from a different package instead of importing (don't know why no import). Probably wanted more control over the generated data. There will probably be more functions added